README.md

```diff
@@ -114,88 +98,3 @@ $ ./scripts/manage.sh selenium
 ```
 
 In addition, other [scripts](scripts/) exist if you want to test just one of the client or server-side components.
-
-## Deployment
-
```

Comment:
Consider leaving the `## Deployment` header here and linking to the deployment README